web/urls.py

```diff
@@ -17,4 +17,5 @@
 	url(r'^privacy/$', views.privacy, name='pages-privacy'),
 	url(r'^project/(?P<project_id>[0-9]+)/$', views.project_detail),
 	url(r'^logout/$', views.logout, name='logout'),
+	url(r'^member_page/$', views.member_page, name='pages-member_page'),
```

Comment:
I think you should use dynamic url here, to create url for each member (for example by username) and in view use this argument to filter your data.
Something like I did with projects and project detail.